recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
   host:
     - python >=3.7
     - pip
+    - setuptools =65.5.1
+    - tomlkit =0.11.6
```

Comment:
```suggestion
    - setuptools >=65.5.1,<65.7.0
    - tomlkit >=0.11.6,<1.0.0
```
Even my suggestions here aren't quite consistent with:
https://github.com/neo4j/neo4j-python-driver/blob/5.3.0/pyproject.toml#L51
But if this works, maybe it's just that the requirements upstream could be loosened.

Reply:
The requirement `setuptools >=65.6,<65.7` is why I didn't merge #54 and was waiting on https://github.com/conda-forge/setuptools-feedstock/pull/297

Reply:
Update: https://github.com/conda-forge/setuptools-feedstock/pull/297 has been merged so we should really follow the upstream constraints.  I have updated my suggestion accordingly.